shoop/configuration.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from shoop.core import cache
+from shoop.core.models import ConfigurationItem
+
+
+SHOP_CONFIGURATION_NAMESPACE = "shop_config"
+
+
+def _get_cache_key(shop):
+    return "%s:%s" % (SHOP_CONFIGURATION_NAMESPACE, shop.pk if shop else "")
```

Comment:
Move this and SHOP_CONFIGURATION_NAMESPACE to bottom. Make the global variable private, i.e. `_SHOP_CONFIGURATION_NAMESPACE`.